modules/social_features/social_event/assets/css/social-event--admin.scss

```diff
@@ -0,0 +1,71 @@
+@mixin bg-block {
+  max-width: 500px;
+  background: #f3f3f3;
+  border-radius: 5px !important;
+
+  label {
+    margin-bottom: .5rem;
+  }
+}
+
+.node-event-form,
+.node-event-edit-form {
+  .field--name-field-event-date,
+  .field--name-field-event-date-end {
+    .card {
+      box-shadow: none;
+      border-radius: 0 !important;
+    }
+
+    .card__title {
+      padding-right: 0;
+      padding-left: 0;
+      border: 0;
+      font-size: .875rem;
+    }
+
+    .card__block {
+      @include bg-block;
+
+      .form-inline {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        flex-wrap: wrap;
+
+        > .form-item {
+          flex: 0 0 100%;
+          max-width: 100%;
+
+          &:first-child {
+            @media(min-width: 600px) {
+              flex: 0 0 245px;
+              max-width: 245px;
+            }
+          }
+
+          &:last-child {
+            @media(min-width: 600px) {
+              flex: 0 0 195px;
+              max-width: 190px;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .field--type-address {
```

Comment:
@agami4 can you put this in the global styling so it is applied on every form where the address field is used?

Reply:
I've moved general styles for location block to socialbase theme.